docs/solidity-by-example.rst

```diff
@@ -89,11 +89,10 @@ of votes.
         function giveRightToVote(address voter) public {
             // If the argument of `require` evaluates to `false`,
             // it terminates and reverts all changes to
-            // the state and to Ether balances. It is often
-            // a good idea to use this if functions are
-            // called incorrectly. But watch out, this
-            // will currently also consume all provided gas
-            // (this is planned to change in the future).
+            // the state and to Ether balances. 
+            // require wont consume gas.
```

Comment:
Please use proper English, capitalize the first word in a sentence and and avoid the use of contractions.

Perhaps it is also better to state that this consumes all gas in old EVM versions but not anymore.

Reply:
I'm not used to capitalize a word when it's the name of a function, but you're right, it does not look so good. I update with the suggested statement.